### DISCUSSION TITLE
How to Uninstall Micromamba on Windows?

### DISCUSSION THREAD

**Discussion by navneethc** [mamba-org/micromamba-docker#463] [Q&A] [answered]
I apologise in advance if this is not the appropriate forum, but this is closest repo I could find for a question about Micromamba.

What are the steps to uninstall Micromamba from Windows 10? On a laptop, not Docker.

None of usual sources, including the docs, SO, and LLMs with a chat interface seem to have the answer.

**Comment by wholtz** [comment, marked as answer]
For non-docker questions, please ask at https://github.com/mamba-org/mamba
